Mlem/Models/Content/Post/PostModel+MenuFunctions.swift

```diff
@@ -141,7 +141,41 @@ extension PostModel {
             }
         }
 
+#if DEBUG
+        functions.append(
+            buildDeveloperMenu(
+                editorTracker: editorTracker,
+                postTracker: postTracker
+            )
+        )
+#endif
+        
         return functions
     }
     // swiftlint:enable function_body_length
+    
+#if DEBUG
+    private func buildDeveloperMenu(
+        editorTracker: EditorTracker,
+        postTracker: StandardPostTracker?
+    ) -> MenuFunction {
+        MenuFunction.childMenu(
+            titleKey: "Developer Menu",
+            children: [
+                .standardMenuFunction(
+                    text: "Toggle Read State",
+                    imageName: "",
```

Comment:
Shuts up the "image not found in symbol set" warning, plus an excuse to use a niche SFSymbol
```suggestion
                    imageName: "book.and.wrench",
```

Reply:
Use all the sf symbols!